BillTrack.Api/Features/Departments/Endpoints/CreateDepartment.cs

```diff
@@ -0,0 +1,33 @@
+using BillTrack.Core.Contracts.Department;
+using BillTrack.Core.Interfaces.Services;
+using BillTrack.Domain.Entities;
+using FastEndpoints;
+using IMapper = AutoMapper.IMapper;
+
+namespace BillTrack.Api.Features.Departments.Endpoints;
+
+public class CreateDepartment : Endpoint<DepartmentRequest, DepartmentResponse>
+{
+    private readonly IMapper _mapper;
+    private readonly IWebApiService _webApiService;
+    
+    public CreateDepartment(IMapper mapper, IWebApiService webApiService)
+    {
+        _mapper = mapper;
+        _webApiService = webApiService;
+    }
+    
+    public override void Configure()
+    {
+        Post("departments");
+        AllowAnonymous();
```

Comment:
our web application - is for inner usage. So a non-authorized user should not access any endpoint.
We will require authentication. And definitely not allowing anonymous requests. It could be removed here or in further tasks.